Compiler/Util/List.mo

```diff
@@ -6221,7 +6221,7 @@ public function deletePositions<T>
 protected
   list<Integer> sorted_pos;
 algorithm
-  sorted_pos := sort(inPositions, intGt);
+  sorted_pos := sortedUnique(sort(inPositions, intGt), intEq);
```

Comment:
Would a List.sortIntN(inPositions, max(inPositions)) be more efficient?

Reply:
`sortIntN({1000000}, 1000000)` uses in the order of a million operations to sort a single element. It seems like it would have a bad worst case behaviour.